### PR TITLE
Dragable metadata.

### PR DESCRIPTION
**Pull request by martinnj**
Implements #1372

### CODE REVIEW

**Comment by Postremus** [comment]
I was just going to implement it. Thanks!

**Comment by martinnj** [comment]
No problem.

File: CKAN.sln [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 2012
```
Comment (Postremus) [inline CKAN.sln]: The changes in the solution file are not needed as far as i can see.

File: GUI/Main.cs [hunk 2]
```diff
@@ -1042,6 +1042,42 @@ private void FocusMod(string key, bool exactMatch)
                 this.AddStatusMessage("Not found");
             }
         }
+
+        private void selectAllInstalledModsToolStripMenuItem_Click(object sender, EventArgs e)
```
Comment (Postremus) [inline GUI/Main.cs]: Please remove your deselect / select Feature form this branch (by removing the commits). Just to kepp things clean.

File: Tests/Core/Relationships/RelationshipResolver.cs [hunk 3]
```diff
@@ -1,819 +1,819 @@
-﻿using System;
```
Comment (Postremus) [inline Tests/Core/Relationships/RelationshipResolver.cs]: No changes done in this file, maybe formating changes?

**Comment by Postremus** [comment]
The feature itself works. Just a minor thing: could you please make the grabable area visible?

**Comment by Postremus** [comment]
fixes part of #940

**Pull request closed without merging**